src/abstractdataframe/show.jl

```diff
@@ -569,6 +569,7 @@ end
 #' showcols(df)
 function showcols(io::IO, df::AbstractDataFrame, all::Bool = false,
                   values::Bool = true) # -> Nothing
+    Base.depwarn("showcols has been deprecated. Use `describe(df, stats = [:eltype, :nmissing, :first, :last]` instead", :showcols)
```

Comment:
The standard approach for deprecations is to completely remove this function and add this to deprecated.jl:
```julia
@deprecate showcols(df::AbstractDataFrame, all::Bool=false, values::Bool=true) describe(df, stats = [:eltype, :nmissing, :first, :last])
@deprecate showcols(io::IO, df::AbstractDataFrame, all::Bool=false, values::Bool=true) show(io, describe(df, stats = [:eltype, :nmissing, :first, :last]), all)
```

Reply:
Okay, so showcols no longer exists and the result just calls `describe`?

Reply:
I deleted all the `showcols` code and added the above code to `deprecated`. I think this works.

Reply:
I also purged the tests of `showcols`. I hope this wasn't too extreme